simple_salesforce/api.py

```diff
@@ -141,6 +151,14 @@ def __init__(
         self.apex_url = ('https://{instance}/services/apexrest/'
                          .format(instance=self.sf_instance))
 
+        # Replace default formats with the custom URL
```

Comment:
The default `base_url` and `apex_url` assignments should be in an `else` block here.